style.css

```diff
@@ -1 +1,294 @@
-/* 본인의 디자인 감각을 최대한 발휘해주세요! */
+@font-face {
+  font-family: 'Pretendard-Regular';
+  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff')
+    format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
+/* reset.css 설정 시작 */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  font-family: 'Pretendard-Regular';
+}
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  line-height: 1.5;
+  -webkit-font-smoothing: antialiased;
+}
+img,
+picture,
+video,
+canvas,
+svg {
+  display: block;
+  max-width: 100%;
+}
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  overflow-wrap: break-word;
+}
+#root,
+#__next {
+  isolation: isolate;
+}
+/* reset.css 설정 완료 */
+/* 스크롤 바 관련 UI를 제거하는 속성 원하는 요소에 scroll-box 클래스를 부여하면 됨 */
+.scroll-box {
+  -ms-overflow-style: none;
+}
+
+.scroll-box::-webkit-scrollbar {
+  display: none;
+}
+
+body {
+  min-height: 100vh;
+  max-height: 100vh;
+  background-image: url('./assets/images/background-image.jpg');
+  background-position: center;
+  display: flex;
+  justify-content: center;
+  padding: 10% 0;
+  overflow: hidden;
+  position: relative;
+  isolation: isolate;
+}
+
+body::after {
+  content: '';
+  position: absolute;
+  background: white;
+  z-index: -1;
+  inset: 0;
+  opacity: 0.2;
+}
+
+.focus-container {
+  width: 70%;
+  background-color: rgba(188, 219, 173, 1);
+  display: flex;
+  flex-direction: column;
+  overflow-y: hidden;
+  border-radius: 10px;
+  height: 80vh;
+}
+
+@media screen and (max-width: 768px) {
+  body {
+    padding: 0;
+  }
+  .focus-container {
+    width: 100%;
+    height: 100dvh;
+  }
+}
+
+.input-section {
+  flex-grow: 2;
+  flex-shrink: 0;
+  min-height: calc(100% * 2 / 7);
+  display: flex;
+  flex-direction: column;
+}
+
+.appName-Date-section {
+  flex-grow: 1;
+  flex-shrink: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50%;
+}
+
+.appName-Date-section > span {
+  font-size: 30px;
+  font-weight: 600;
+  width: 50%;
+  text-align: center;
+}
+
+.date-input {
+  margin-right: 10px;
+  /* 기본 스타일링 제거 */
+  border: none;
+  outline: none;
+  height: 35px;
+  text-indent: 5px;
+  border-radius: 5px;
+  padding-right: 5px;
+  font-weight: 500;
+}
+
+@media screen and (max-width: 768px) {
+  .date-input {
+    font-size: 10px;
+    color: black;
+  }
+}
+
+.input-box-section {
+  flex-grow: 1;
+  flex-shrink: 0;
+  height: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 20px;
+}
```

Comment:
할 일을 길게 입력하면 양 쪽에 공백이 없어서 이런식으로 padding 추가해주는 것도 좋을 것 같습니다! 
```suggestion
padding : 10px;
```
<img width="596" alt="스크린샷 2024-03-17 오전 8 38 21" src="https://github.com/CEOS-Developers/vanilla-todo-19th/assets/117785708/61ba4dd0-222e-4006-a5fd-6579d042ea88">